src/main/java/io/jenkins/plugins/opentelemetry/OpenTelemetryConfiguration.java

```diff
@@ -103,8 +103,8 @@ public Map<String, String> toOpenTelemetryProperties() {
             properties.put("otel.imr.export.interval", "10ms");
         } else if (this.getEndpoint().isPresent()) {
             this.getEndpoint().ifPresent(endpoint -> { // prepare of Optional.ifPResentOrElse()
-                properties.compute("otel.traces.exporter", (key, oldValue) -> oldValue == null ? "otlp" : oldValue.contains("otlp") ? oldValue : oldValue.concat(",otlp"));
-                properties.compute("otel.metrics.exporter", (key, oldValue) -> oldValue == null ? "otlp" : oldValue.contains("otlp") ? oldValue : oldValue.concat(",otlp"));
+                properties.compute("otel.traces.exporter", (key, oldValue) -> oldValue == null ? "otlp" : "none".equals(oldValue) ? "none" : oldValue.contains("otlp") ? oldValue : oldValue.concat(",otlp"));
```

Comment:
5 alternatives inline expressions are difficult to follow

```suggestion
                properties.compute("otel.traces.exporter", (key, oldValue) -> {
                  if (oldValue == null) {
                    return "otlp"
                   } else if ("none".equals(oldValue)) {
                     return "none"
                   } else if (oldValue.contains("otlp")) {
                     return oldValue
                   } else {
                     return oldValue.concat(",otlp");
                   });
```